### ISSUE TITLE
LspInstall not working Windows

### ISSUE THREAD

**Issue by RefusesNames** [neovim/nvim-lspconfig#359]
- `nvim --version`: NVIM v0.5.0-705-g9f704c88a
- nvim-lsp version(commit hash):
    How can I see that? nvim-lspconfig has the commit `60133c47e0fd82556d7ca092546ebfa8d047466e`
- `:checkhealth` result:
    configuration ok, performance ok, remote plugins ok, clipboard ok, python2 and 3 ok, virtualenv ok, node.js ok, perl and ruby provider missing
- What language server(If the problem is related to a specific language server): All of them (`vimls` as an example)
- Operating system/version: Windows 10 Pro (Version 2004)
- Terminal name/version: PowerShell core (7.1.0-preview.7)
- `$TERM`: xterm-256color

### How to reproduce the problem from neovim startup
```
:LspInstall vimls
```

### Actual behaviour
Message: "Installation requires "sh", "npm", "mkdir"

### Expected behaviour
Installation of the language server

**Comment by tarasyarema** [comment]
I guess the first think to do is look here: https://github.com/neovim/nvim-lspconfig/blob/60133c47e0fd82556d7ca092546ebfa8d047466e/lua/nvim_lsp/util.lua#L378.

It seems like the only system implemented are the ones that have `sh`, `npm` and `mkdir`. I think that this can be easly changed to something like this:

```lua
local cmd = io.popen("powershell -command -", "w")
    local install_script = ([[
    mkdir -p "{{install_dir}}"
    cd "{{install_dir}}"
    && npm init -y
    npm install {{packages}} --no-package-lock --no-save --production
    {{post_install_script}}
    ]]):gsub("{{(%S+)}}", install_params)
    cmd:write(install_script)
    cmd:close()
```

**Comment by lithammer** [comment]
I think the idea is to completely drop the installers. See https://github.com/neovim/nvim-lspconfig/issues/334.

**Comment by tarasyarema** [comment]
I was just reading that issue jaja

**Comment by mjlbach** [comment]
Closed by #498
